x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/update_rules_schema.test.ts

```diff
@@ -1243,4 +1243,101 @@ describe('create rules schema', () => {
       'child "severity" fails because ["severity" must be one of [low, medium, high, critical]]'
     );
   });
+
+  describe('note', () => {
+    test('You can set note to a string', () => {
+      expect(
+        updateRulesSchema.validate<Partial<RuleAlertParamsRest>>({
+          rule_id: 'rule-1',
+          output_index: '.siem-signals',
+          risk_score: 50,
+          description: 'some description',
+          from: 'now-5m',
+          to: 'now',
+          index: ['index-1'],
+          name: 'some-name',
+          severity: 'low',
+          interval: '5m',
+          type: 'query',
+          references: ['index-1'],
+          query: 'some query',
+          language: 'kuery',
+          max_signals: 1,
+          note: '# some documentation title',
+        }).error
+      ).toBeFalsy();
+    });
+
+    test('You can set note to an empty string', () => {
+      expect(
+        updateRulesSchema.validate<Partial<RuleAlertParamsRest>>({
+          rule_id: 'rule-1',
+          output_index: '.siem-signals',
+          risk_score: 50,
+          description: 'some description',
+          from: 'now-5m',
+          to: 'now',
+          index: ['index-1'],
+          name: 'some-name',
+          severity: 'low',
+          interval: '5m',
+          type: 'query',
+          references: ['index-1'],
+          query: 'some query',
+          language: 'kuery',
+          max_signals: 1,
+          note: '',
+        }).error
+      ).toBeFalsy();
+    });
+
+    // Note: If you're looking to remove `note`, omit `note` entirely
```

Comment:
@FrankHassanabad this might be overkill, bu thought it might be helpful in trying to be clear why it's not nullable here.

Reply:
This comment is fine.